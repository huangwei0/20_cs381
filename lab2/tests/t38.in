debug off
stopwords on
load tracks.txt
preprocess
mcw were
mcw you
mcw expecting
mcw something
mcw else