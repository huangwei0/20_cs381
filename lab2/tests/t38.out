you
are
fly
about
is
